tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.psm1

```diff
@@ -838,26 +829,33 @@ function Get-AzureRmProfile
     if ($ListAvailable.IsPresent)
     {
       Write-Verbose "Getting all the profiles available for install"
-      Get-ProfilesAvailable $ProfileMap
+      foreach ($profile in ($ProfileMap | get-member -MemberType NoteProperty).Name)
+      {
+        $profileObj = $ProfileMap.$profile
+        $profileObj | Add-Member -MemberType NoteProperty -Name "ProfileName" -Value $profile
```

Comment:
Never mind, what I meant was that the two commands where you are piping to ```Add-Member``` in addition to the line where $profileObj is returned.  SInce Add-Member produces no output, you canignore